plugins/woocommerce-admin/client/tasks/fills/PaymentGatewaySuggestions/index.js

```diff
@@ -213,19 +250,42 @@ export const PaymentGatewaySuggestions = ( { onComplete, query } ) => {
 							wcPay.push( gateway );
 						} else if ( gateway.is_offline ) {
 							offline.push( gateway );
-						} else {
+						} else if ( gateway.enabled ) {
```

Comment:
Looks like the `PaymentGatewaySuggestions` component is more complicated now because we have many rules here. 😂  Maybe we could create a following up issue to refactor it?

To refactor it, I think we could refactor the arrow functions`useMemo()` by moving them outside the `PaymentGatewaySuggestions` component or event in a separate file.  So we can also test them individually.

For example:

Reply:
Great suggestion! I'll do this in a follow-up 👍